packages/@aws-cdk/pipelines/README.md

```diff
@@ -761,6 +761,37 @@ leading NPM 6 reading that same file to not install all required packages anymor
 Make sure you are using the same NPM version everywhere, either downgrade your
 workstation's version or upgrade the CodeBuild version.
 
+### Cannot connect to the Docker daemon at unix:///var/run/docker.sock
+
+If, in the 'Synth' action (inside the 'Build' stage) of your pipeline, you get an error like this:
+
+```console
+stderr: docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.   
+See 'docker run --help'. 
+```
+
+It means that the AWS CodeBuild project for 'Synth' is not configured to run in privileged mode, 
+which prevents Docker builds from happening.
```

Comment:
Maybe also mention here why this might be happening. Something like:

> This typically happens if you use a CDK construct that bundles asset using tools run via Docker, like `aws-lambda-nodejs`, `aws-lambda-python`, `aws-lambda-go` and others.

Also good to mention somewhere:

> After turning on `privilegedMode: true`, you will need to do a one-time manual `cdk deploy` of your pipeline to get it going again (as with a broken 'synth' the pipeline will not be able to selfupdate to the right state).